hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemOauth.java

```diff
@@ -146,6 +146,8 @@ public void testBlobDataReader() throws Exception {
       abfsStore.openFileForWrite(EXISTED_FILE_PATH, fs.getFsStatistics(), true);
     } catch (AbfsRestOperationException e) {
       assertEquals(AzureServiceErrorCode.AUTHORIZATION_PERMISSION_MISS_MATCH, e.getErrorCode());
+    } finally {
+      abfsStore.close();
```

Comment:
Use IOUtils.cleanupWithLogger() here as well. Or you can just say this code is old.

Reply:
.close() does this already.